cosmos-script-runner-api/app/models/running_script.rb

```diff
@@ -477,7 +479,9 @@ def stop?
     @stop
   end
 
-  def clear_prompt()
+  def clear_prompt
+    # Allow things to continue once the prompt is cleared
+    @go = true
```

Comment:
This is what allows the current prompt which is sitting in the wait_for_go_or_stop method to continue at which point the prompt method (implemented in define_method) loops back around and does it again. Unfortunately I think there's a race condition with the frontend which is still present.